src/distributed-transaction/introduction.md

```diff
@@ -0,0 +1,13 @@
+# Introduction
+
+TiKV is a distributed transactional key-value database, transaction is a foundational function for TiKV. This chapter we will talk about some concepts about distributed transaction and some also provide some details about distributed transaction implementation in TiKV.
+
+A database transaction, by definition, must be atomic, consistent, isolated and durable. Database practitioners often refer to these properties of database transactions using the acronym ACID.
+
+Transactions provide an "all-or-nothing" proposition, stating that each work-unit performed in a database must either complete in its entirety or have no effect whatsoever. Further, the system must isolate each transaction from other transactions, results must conform to existing constraints in the database, and transactions that complete successfully must get written to durable storage.
+
+A distributed transaction is a database transaction in which two or more network hosts are involved. Usually, hosts provide transactional resources, while the transaction manager is responsible for creating and managing a global transaction that encompasses all operations against such resources. Distributed transactions, as any other transactions, must have all four ACID properties.
+
+A common algorithm for ensuring correct completion of a distributed transaction is the two-phase commit (2PC). 
+
+TiKV adopts google Percolator Transaction model, it is a variant of 2PC.
```

Comment:
You can drop the `it is`. 

`TiKV adopts google Percolator Transaction model, a variant of 2PC.`